models.py

```diff
@@ -37,10 +38,16 @@ def serialize(self):
             "is_admin": self.is_admin,
         }
 
-
 MARKER_TYPE_ACCIDENT = 1
 MARKER_TYPE_DISCUSSION = 2
 
+        
+class Subscriber(Base):
```

Comment:
Why does this require a new class and not simply a boolean column in the User class?